collective/searchandreplace/searchreplaceutility.py

```diff
@@ -389,7 +397,9 @@ def setTextField(obj, fieldname, text):
                 text = RichTextValue(text)
             else:
                 text = RichTextValue(text, old.mimeType, old.outputMimeType)
-        field.set(obj, text)
+        if ITuple.providedBy(field) and ITextLine.providedBy(field.value_type):
+            text = tuple(text.split('\n'))
+        setattr(field.interface(obj), field.__name__, text)
```

Comment:
I do not have deep dexterity experience.

@mauritsvanrees, does the change from `field.set` to `setattr` misses something ?

Reply:
The new code seems okay, and a bit better in case the field is from a behavior that does not use simple setting of an attribute, but does more, like in `plone.app.event` (although that does not contain LinesFields AFAIK).